modules/link-in-bio/module.php

```diff
@@ -44,17 +49,36 @@ public function __construct() {
 	/**
 	 * Register styles.
 	 *
-	 * At build time, Elementor compiles `/modules/link-in-bio/assets/scss/frontend.scss`
-	 * to `/assets/css/widget-link-in-bio.min.css`.
+	 * At build time, Elementor compiles `/modules/link-in-bio/assets/scss/widgets/*.scss`
+	 * to `/assets/css/widget-*.min.css`.
 	 *
 	 * @return void
 	 */
 	public function register_styles() {
-		wp_register_style(
-			'widget-link-in-bio',
-			$this->get_css_assets_url( 'widget-link-in-bio', null, true, true ),
-			[ 'elementor-frontend' ],
-			ELEMENTOR_VERSION
-		);
+		$widget_styles = self::get_widget_style_list();
+		$has_custom_breakpoints = Plugin::$instance->breakpoints->has_custom_breakpoints();
+
+		foreach ( $widget_styles as $widget_style_name => $widget_has_custom_breakpoints ) {
+			$custom_breakpoints = $widget_has_custom_breakpoints ? $has_custom_breakpoints : false;
+
+			wp_register_style(
+				$widget_style_name,
+				$this->get_css_assets_url( $widget_style_name, null, true, $custom_breakpoints ),
+				[ 'elementor-frontend' ],
+				$custom_breakpoints ? null : ELEMENTOR_VERSION
+			);
+		}
+	}
+
+	private function get_widget_style_list() {
+		return [
+			'widget-link-in-bio' => self::WIDGET_HAS_CUSTOM_BREAKPOINTS, // TODO: Remove in v3.27.0 [ED-15717]
+			'widget-link-in-bio-base' => self::WIDGET_HAS_CUSTOM_BREAKPOINTS,
+			'widget-link-in-bio-var-2' => self::WIDGET_HAS_NO_CUSTOM_BREAKPOINTS,
```

Comment:
We can use:

```php
'widget-link-in-bio-var-2' => ! self::WIDGET_HAS_CUSTOM_BREAKPOINTS,
```

Reply:
@rami-elementor Amazing! I have implemented it.